frontend/src/components/PostContent.js

```diff
@@ -0,0 +1,102 @@
+import React, { useMemo } from 'react'
+
+import { Link, useLocation } from 'react-router-dom'
+import { parseISO } from 'date-fns'
+
+import { timeFormat } from '../utils'
+import { useIsPostPublic } from '../utils/customHooks/useIsPostPublic'
+import useUser from '../utils/customHooks/useUser'
+
+import Avatar from './Avatar'
+import CopyPostURL from './CopyURL'
+import Markdown from './Markdown'
+import PostCategories from './PostCategories'
+import ReactionBar from './ReactionBar'
+import TextBlock from './TextBlock'
+import { Transition } from './Transition'
+import UserPostMenu from '../authenticated/UserPostMenu'
+
+const PostContent = ({ animationDelay, post, review, userMenu }) => {
+  const { pathname } = useLocation()
+  const user = useUser()
+
+  const isPublic = useIsPostPublic(post.isPublic)
+  const parsed = parseISO(post.createdAt)
+  const date = timeFormat(parsed)
+
+  const isPostOwner = useMemo(() => {
+    if (user && post) {
+      return user.uuid === post.author.uuid
+    }
+  }, [user, post])
+
+  let title
+  if (pathname === '/search') {
+    title = <TextBlock value={post.title} />
+  } else {
+    title = post.title
+  }
```

Comment:
could be `const title = pathname === 'search' ? <TextBlock value={post.title} /> : post.title`